internal/app/backend/backend_service.go

```diff
@@ -98,10 +101,15 @@ func (s *backendService) FetchMatches(ctx context.Context, req *pb.FetchMatchesR
 		return nil, err
 	}
 
+	err = doFetchMatchesAddIgnoredTickets(ctx, s.store, results)
```

Comment:
Is this helper for testability? otherwise it may not be necessary? 

If you keep it, you may use short form error check - 
if err = do... ; err != nil {

}